app/src/components/wizard-steps.tsx

```diff
@@ -26,40 +28,59 @@ export default function WizardSteps({
 }) {
   const orientation: "horizontal" | "vertical" | undefined = useBreakpointValue(
     { base: "vertical", md: "horizontal" },
-    { fallback: "md" }
+    { fallback: "md" },
   );
   const gap: "0" | undefined = useBreakpointValue(
     { base: "0", md: undefined },
-    { fallback: "md" }
+    { fallback: "md" },
   );
 
-  return (
-    <Stepper
```

Comment:
I think it would be good if you could add a new component here instead of modifying the existing WizardSteps component. I think it is also used on the add data page, so let's make sure we're not breaking that one by applying these changes.